build-aux/snap/snapcraft.yaml

```diff
@@ -285,6 +286,8 @@ parts:
         ) > "${CRAFT_PART_INSTALL}/usr/lib/snapd/dirty-git-tree-info.txt"
       fi
 
+      find "${CRAFT_PART_INSTALL}/usr/lib/snapd" "${CRAFT_PART_INSTALL}/usr/bin" -type f -exec strip {} ";"
```

Comment:
Is this safe for Go?

Reply:
Well dpkg does it, so...

Reply:
(it might be debhelper), but for sure the binaries in the snapd snap currently have no debuginfo.

Reply:
@zyga this has been safe for go for many years now.